package.json

```diff
@@ -42,8 +39,7 @@
     "eslint": "^9.3.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-prettier": "^5.1.3",
-    "husky": "^9.0.11",
-    "lint-staged": "^15.2.4",
+    "lefthook": "^1.6.16",
     "pinst": "^3.0.0",
```

Comment:
should `pinst` be removed?

Reply:
Ohh shit missed it! removed at https://github.com/scaffold-eth/create-eth/pull/51/commits/23462beee6ee80ccec9ca840bfaf5f603806d872 Thanks !